eclipse-che/che-plugin-registry
# chore: add memoryRequest for plugins sidecars

Signed-off-by: Vitaliy Gulyy <vgulyy@redhat.com>



### What does this PR do?
Adds memory request in 20 megabytes for the plugin sidecars.

PR with pull-request check for memory and cpu limitations https://github.com/eclipse-che/che-plugin-registry/pull/1022

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/16685

### How to test this PR?

 
A plugin registry with memory requests is temporary published to https://vgulyy-plugin-registry.surge.sh/v3/
 
1. create any workspace, install recommended plugins
2. stop the workspace, go to devfile on the dashboard
3. add registry URL to all the plugins to load them from temporary plugin registry
```
registryUrl: 'https://vgulyy-plugin-registry.surge.sh/v3'
```
4. start workspace, check how plugin works

### Tested on:
- local minikube
- workspaces.openshift.com
- openshift 4.7 cluster on RHPDS

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
